README.md

```diff
@@ -688,13 +688,15 @@ Output:
 ---
 
 Both the `field` and the global Blueprinter Configuration supports `:if` and `:unless` options that can be used to serialize fields conditionally.
+By default, field-level conditions override global conditions. By enabling `roll_up_conditions`, BOTH field-level and global conditions must be met in order for the field to be serialized.
 
 ### Global Config Setting - if and unless
 
 ```ruby
 Blueprinter.configure do |config|
   config.if = ->(field_name, obj, _options) { !obj[field_name].nil? }
   config.unless = ->(field_name, obj, _options) { obj[field_name].nil? }
+  config.roll_up_conditions = true
```

Comment:
@lessthanjacob suggested `roll_up_conditions` is a bit unclear. How about `enforce_all_conditions`?

Reply:
Thanks for the ping! You caught me in the middle of writing a quick comment 😆

The desired behavior makes sense to me, but the name of the configuration setting gives me slight pause. Ideally it's something that you can set and forget, but I'd challenge us to think of a name that describes the functionality a bit more explicitly! 

I could be on board with `enforce_all_conditions`. We could also codify the default behavior in the configuration, and expect that we set it to `false` to get the behavior detailed in this PR (e.g. `field_level_condition_overrides_global`).